exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/using_a_script_languages_container.ipynb

```diff
@@ -0,0 +1,704 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "27352b75-823d-40a1-b8df-d33218fd7de6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from exasol.nb_connector import slct_manager\n",
+    "slctmanager = slct_manager.SlctManager(ai_lab_config)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9fba4c9e-4a1b-4f84-a9a6-f05810aa8922",
+   "metadata": {},
+   "source": [
+    "### Import some utility functions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c554d698-d5a2-4f9b-846a-f606a3f183b7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from exasol.nb_connector.utils import upward_file_search\n",
+    "\n",
+    "# This NB may be running from various locations in the NB hierarchy.\n",
+    "# Need to search for the styles NB from the current directory upwards.\n",
+    "\n",
+    "%run {upward_file_search('utils/ui_styles.ipynb')}\n",
+    "%run {upward_file_search('utils/popup_message_ui.ipynb')}\n",
+    "%run ./utils/file_system_ui.ipynb\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c88668d2-83e5-4812-a05c-936912068f03",
+   "metadata": {},
+   "source": [
+    "## Use flavor as is\n",
+    "\n",
+    "Currently used flavor is:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "617380fa-6b14-40a5-bffd-ca8fed37950c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "slct_manager.REQUIRED_FLAVOR"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d496fe3d-90ac-4d3e-86e0-bef8ecaf8561",
+   "metadata": {},
+   "source": [
+    "### Export the flavor\n",
+    "You could run the same on the command line with the `exaslct` tool:\n",
+    "```\n",
+    "cd <slc_dir>\n",
+    "./exaslct export --flavor-path flavors/<flavor_path>\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "20665390-4f31-48db-ab78-249a1ef629e9",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "slctmanager.export()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "deb31b95-f615-45c1-b47a-e8e1b0fb22c9",
+   "metadata": {},
+   "source": [
+    "#### Check the result"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fc142500-d855-4112-bba2-ed3395889b9b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"Directory content:\")\n",
+    "show_directory_content(slctmanager.working_path.export_path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "33d76239-60fd-430f-b7e2-fca74c2e0871",
+   "metadata": {},
+   "source": [
+    "### What to do if something doesn't work?\n",
+    "\n",
+    "During the build it can happen that external package repositories might not be available or something is wrong on your machine where you run the build. For these cases, exaslct stores many logs to identify the problem.\n",
```

Comment:
```suggestion
    "During the build it can happen that external package repositories might not be available or something is wrong on your machine running the build. `exaslct` therefore provides extensive logging facilities for analyzing such problems.\n",
```